server/insights/src/main/java/org/infinispan/server/insights/logging/Log.java

```diff
@@ -43,17 +41,16 @@ public interface Log extends BasicLogger {
    @Message(value = "Remote Insights reporting", id = 32005)
    void insightsEnabled();
 
-   @Message(value = "Error configuring Red Hat Insight client", id = 32006)
-   CacheConfigurationException insightsConfigurationError();
-
    @LogMessage(level = ERROR)
```

Comment:
Actually I have a different idea. I would implement the following behavior:
If the Insights module detects that the OPT_OUT is enabled, there is no reason to run the Insights client, in this case I think that it would be better to present, yes, an INFO message to the user, and return.
On the other hand I would use the WARN severity here, in order to alert the user properly in case of an expected error. What do you think about it?